runtime/Cargo.toml

```diff
@@ -19,4 +19,5 @@ thiserror = { workspace = true }
 bytes = { workspace = true }
 tokio-util = { version = "0.7", features = ["codec"] }
 sha2 = { workspace = true }
-governor = { workspace = true }
\ No newline at end of file
+governor = { workspace = true }
+prometheus-client = "0.22.3"
```

Comment:
There is a pinned version in the workspace for this one. Would you mind using `prometheus-client = { workspace = true }`?

Reply:
no problem